composer.json

```diff
@@ -28,23 +28,23 @@
     ],
     "require": {
         "php": "^5.4 || ^7.0",
-        "symfony/config": "^2.3 || ^3.0",
-        "symfony/dependency-injection": "^2.3 || ^3.0",
-        "symfony/event-dispatcher": "^2.3 || ^3.0",
-        "symfony/form": "^2.3 || ^3.0",
-        "symfony/http-foundation": "^2.3 || ^3.0",
-        "symfony/http-kernel": "^2.3 || ^3.0",
-        "symfony/options-resolver": "^2.3 || ^3.0"
+        "symfony/config": "~2.8|~3.0|~4.0",
```

Comment:
Please write the requirements in this way: ^2.3 || ^3.0 || ^4.0

Reply:
We still maintain 2.3?

Reply:
yes